src/pages/index.js

```diff
@@ -40,89 +50,222 @@ const enableValidation = (config) => {
-//cardAddFormValidator.enableValidation();
+/* -------------------------------------------------------------------------- */
+/*                                   Profile                                  */
+/* -------------------------------------------------------------------------- */
 /* -------------------------------- UserInfo -------------------------------- */
-const profileInfo = new UserInfo(".profile__name", ".profile__description");
-/* ------------------------------ PopupWithForm ----------------------------- */
+const profileInfo = new UserInfo(
+  ".profile__name",
+  ".profile__description",
+  ".profile__avatar"
+);
+
+api
+  .getUserInfo()
+  .then((res) => {
+    profileInfo.setUserInfo({ name: res.name, description: res.about });
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+/* ------------------------------ ProfilePopup ----------------------------- */
 const profileEditPopup = new PopupWithForm({
   popupSelector: "#profile-edit-modal",
   handleFormSubmit: handleProfileEditSubmit,
 });
 profileEditPopup.setEventListeners();
-const cardAddPopup = new PopupWithForm({
-  popupSelector: "#card-add-modal",
-  handleFormSubmit: handleCardAddSubmit,
-});
-cardAddPopup.setEventListeners();
-/* ----------------------------- PopupWithImage (Card)----------------------------- */
-const cardImagePopup = new PopupWithImage("#image-modal");
-cardImagePopup.setEventListeners();
-/* --------------------------------- Section -------------------------------- */
-const cardList = new Section(
-  {
-    items: initialCards,
-    renderer: renderCard,
-  },
-  ".cards__list"
-);
-cardList.renderItems();
 
-/* -------------------------------------------------------------------------- */
-/*                                  Functions                                 */
-/* -------------------------------------------------------------------------- */
-/* --------------------------------- Profile -------------------------------- */
 function fillProfileForm() {
   const userData = profileInfo.getUserInfo();
   profileTitleInput.value = userData.name;
   profileDescriptionInput.value = userData.job;
 }
-/* ---------------------------------- Card ---------------------------------- */
 
-function handleImageClick(name, link) {
-  cardImagePopup.open(name, link);
+function handleProfileEditSubmit(userData) {
+  profileEditPopup.renderLoading(true);
+  api
+    .editUserInfo(userData)
+    .then((res) => {
+      profileInfo.setUserInfo({ name: res.name, description: res.about });
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      profileEditPopup.renderLoading(false);
+    });
+  profileEditPopup.close();
 }
 
+profileEditButton.addEventListener("click", () => {
+  fillProfileForm();
+  formValidators[profileEditForm.getAttribute("id")].resetValidation();
+  profileEditPopup.open();
+});
+
+/* ------------------------------- avatarPopup ------------------------------ */
+api
+  .getUserAvatar()
```

Comment:
`api.getUserAvatar()` is not needed in your project because you get everything when you call `api.getUserInfo()`